core/src/main/java/org/fenixedu/messaging/domain/MessageTask.java

```diff
@@ -17,5 +16,6 @@ public void runTask() {
             dispatched++;
         }
         taskLog("Dispatched %d messages\n", dispatched);
+        MessagingSystem.deleteOldMessages();
```

Comment:
Oh, I see that if any person queues 11 messages, they'll never see if the first one was sent. this to can trigger the Ticketmageddon. Can we ever be safe?! Which do you prefer?